nautobot/core/tests/nautobot_config.py

```diff
@@ -26,3 +27,37 @@
 ]
 
 SECRET_KEY = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+
+def is_truthy(arg):
+    """Convert "truthy" strings into Booleans.
+    Examples:
+        >>> is_truthy('yes')
+        True
+    Args:
+        arg (str): Truthy string (True values are y, yes, t, true, on and 1; false values are n, no,
+        f, false, off and 0. Raises ValueError if val is anything else.
+    """
+    if isinstance(arg, bool):
+        return arg
+    return bool(strtobool(str(arg)))
+
+
+RQ_QUEUES = {
```

Comment:
It's not obvious to me how or why this works - we're setting a different `DB` number for the "default" and "check_releases" queues, but `rqworker` just listens on the queues by name, not by number. Does this work only because we're still running `nautobot-server rqworker` using the (non-testing) `nautobot_config.py` while we're running `nautobot-server test` using this (testing) `nautobot_config.py`, so their configs don't match up?

Some explanatory comments here would be useful. :-)

Reply:
That is exactly right, you could also just stop the nautobot-server rqworker, but then you have to remember to start it back up.  I will add some comments here.